### DISCUSSION TITLE
Emails and api key

### DISCUSSION THREAD

**Discussion by ArnoldsAizkalns** [vercel/community#2671] [Help]
### Summary

I have a problem using SendGrid with Vercel.
Initially, I set up and connected SendGrid on my local server, and it works perfectly.
But when I deploy using Vercel, the submitted forms do not reach the email!
Vercel logs show a status code of 200, indicating that everything was sent successfully!
However, SendGrid does not seem to record the email sent from Vercel. Has anyone encountered such an issue?

### Steps to Reproduce

Just if someone have experience in issue like that

**Comment by smaeda-ks** [comment]
hey @ArnoldsAizkalns, please make sure you're using `await` properly: https://vercel.com/guides/serverless-functions-and-smtp#using-smtp

See also:
- https://docs.aws.amazon.com/lambda/latest/dg/nodejs-handler.html
- https://levelup.gitconnected.com/avoiding-the-pitfalls-of-async-node-js-functions-in-aws-lambda-941220582e7a
